.github/workflows/rust.yml

```diff
@@ -91,3 +91,61 @@ jobs:
     steps:
     - uses: actions/checkout@v4
     - uses: EmbarkStudios/cargo-deny-action@v2
+
+  test-android:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        include:
+          - target: x86_64-linux-android
+            emulator-arch: x86_64
+            # Note that x86_64 image is only available for API 21+. See
+            # https://github.com/ReactiveCircus/android-emulator-runner?tab=readme-ov-file#configurations.
+            api-level: 21
+          - target: i686-linux-android
+            emulator-arch: x86
+            api-level: 19
```

Comment:
Uses Android API level 19 as suggested in https://github.com/quinn-rs/quinn/issues/1503#issuecomment-2285989144 by @link2xt.